packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.component.tsx

```diff
@@ -72,15 +103,20 @@ export const FilesManagerPanel: FunctionComponent<FilesManagerPanelProps> = ({
                 searchValue={searchValue}
                 onSearchValueChange={onSearchValueChange}
               />
-              <ButtonWrapper>
-                <Button
-                  data-testid={FilesManagerPanelTestIds.Button}
-                  displayStyle={DisplayStyle.Primary}
-                  labelMessage={messages.uploadButton}
-                  onClick={onUploadFile}
-                  disabled={disabled}
-                />
-              </ButtonWrapper>
+              {tooManyFiles ? (
+                <ElementWithTooltip Element={uploadButton}>
+                  <StyledTooltipPrimaryContent
+                    description={{
+                      ...messages.tooManyFiles,
+                      values: {
+                        maxFilesCount: filesSlotsHarmonyLimit,
```

Comment:
The variable name could be the same? or to be related? 🤔 

```suggestion
                        filesSlotsHarmonyLimit: filesSlotsHarmonyLimit,
```

or

```suggestion
                        filesSlotsLimit: filesSlotsHarmonyLimit,
```